packages/node_modules/@node-red/nodes/core/function/89-trigger.html

```diff
@@ -74,16 +81,19 @@
         category: 'function',
         color:"#E6E0F8",
         defaults: {
+            name: {value:""},
             op1: {value:"1", validate: RED.validators.typedInput("op1type")},
             op2: {value:"0", validate: RED.validators.typedInput("op2type")},
             op1type: {value:"val"},
             op2type: {value:"val"},
             duration: {value:"250",required:true,validate:RED.validators.number()},
             extend: {value:"false"},
+            second: {value:false},
```

Comment:
I don't think we need both `second` and `outputs` as new properties on the node. Just setting `outputs=2` ought to be enough for both the editor and runtime to know what they need to do.

Reply:
I thought outputs has to be numeric type and a checkbox has to be boolean ? so having two is not a great hardship vs mangling the type on prepare and on save ? Only one is used on the js side

Reply:
Having two inflates the flow file size for absolutely every user of the trigger node, all for the sake of not having to add one line of JavaScript in `oneditprepare`.

Reply:
OK - fixed.